webpack.config.js

```diff
@@ -17,6 +17,10 @@ module.exports = {
       path: require.resolve('path-browserify'),
     },
   },
+  cache: {
+    type: 'filesystem',
+    cacheDirectory: path.resolve(__dirname, '.webpack_cache'),
+  },
```

Comment:
Does this cache work even if GHA cache is not being used?

Reply:
Yes. This is independent.